languagetool-language-modules/nl/src/main/java/org/languagetool/rules/nl/MorfologikDutchSpellerRule.java

```diff
@@ -72,4 +81,46 @@ protected String getProhibitFileName() {
     return "/nl/spelling/prohibit.txt";
   }
 
-}
+  private void initEnglishSpeller() throws IOException {
+    List<String> plainTextDicts = new ArrayList<>();
+    String languageVariantPlainTextDict = null;
+    if (getDataBroker().resourceExists(getSpellingFileName())) {
+      plainTextDicts.add(getSpellingFileName());
+    }
+    for (String fileName : getAdditionalSpellingFileNames()) {
+      if (getDataBroker().resourceExists(fileName)) {
+        plainTextDicts.add(fileName);
+      }
+    }
+    if (getLanguageVariantSpellingFileName() != null && getDataBroker().resourceExists(getLanguageVariantSpellingFileName())) {
+      languageVariantPlainTextDict = getLanguageVariantSpellingFileName();
+    }
+    MorfologikMultiSpeller englishSpeller = new MorfologikMultiSpeller(englishDictFilepath, plainTextDicts, languageVariantPlainTextDict,
+            userConfig, 1, language);
+    setConvertsCase(englishSpeller.convertsCase());
+  }
+
+  @Override
+  public List<RuleMatch> getRuleMatches(String word, int startPos, AnalyzedSentence sentence,
+    List<RuleMatch> ruleMatchesSoFar, int idx,
+    AnalyzedTokenReadings[] tokens) throws IOException {
+    if (tokens[idx].hasPosTag("_FOREIGN_ENGLISH")) {
+      //System.out.println(word + " is assumed English");
+      return Collections.emptyList();
+      /* next step: offer suggestions
+      if (englishSpeller.isMisspelled(word)) {
+        String message = "Het lijkt erop dat dit een foutief gespeld Engels woord is.";
+        String shortMessage = "Mogelijke fout in Engels woord";
+        List<String> suggestions = englishSpeller.getSuggestions(word);
+        RuleMatch ruleMatch = new RuleMatch(this, sentence, startPos, startPos + word.length(),
+          message, shortMessage, suggestions);
+        return Collections.singletonList(ruleMatch);
+      } else {
+        return Collections.emptyList();
+      }
+      */

```

Comment:
I thought that `englishSpeller.isMisspelled(word)` would be used to check the words around `_FOREIGN_ENGLISH` or with this POS tag, but it is not. 
With this configuration, any nonsense word around a word in `ENGLISH_WORDS` will be ignored. Is that what we want?

Reply:
I expected something like: 

```
if (tokens[idx].hasPosTag("_FOREIGN_ENGLISH") && !englishSpeller.isMisspelled(tokens[idx].getToken()) {
  return Collections.emptyList();
}
```